src/MessageHandler/ZipDatasetFilesHandler.php

```diff
@@ -93,7 +93,7 @@ public function __construct(
      */
     public function __invoke(ZipDatasetFiles $zipDatasetFiles)
     {
-        $this->logger->info('Zip worker starting');
+        $this->logger->info('Zip worker starting (ZipDatasetFilesHandler)');
```

Comment:
Might as well change the entire text here, this was a bad cut and paste.

Reply:
Ok, made the text a bit better, and added the PID for the worker in the messages.